demo/admin/src/products/ProductForm.cometGen.ts

```diff
@@ -0,0 +1,68 @@
+import { FormConfig } from "@src/generator/generator";
```

Comment:
Should the config file really be inside the src/ folder?

Reply:
yes; I consider it a "real" source file. And this allows us to put the generated files right next to it.